OasisDEX/oasis-borrow
# Dedicated method for automation sidebar copies

# [Dedicated method for automation sidebar copies](https://app.shortcut.com/oazo-apps/story/5774/dedicated-methods-for-handle-automation-sidebar-copies)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- prepared dedicated method for handling copies within automation features sidebars
- adjusted old methods which will stay for other features (removed automation flows handling from them)
  
## How to test 🧪
  <Please explain how to test your changes>

- test all flows within automation scope and pay attention to copies inside sidebar
- test stop loss setup during opening flow
- few test with opening / managing borrow and multiply vault to make sure that other copies are not broken
